nflow-explorer/package.json

```diff
@@ -4,58 +4,58 @@
   "license": "EUPL-1.1",
   "browsers": "last 1 version",
   "dependencies": {
-    "lodash": "4.17.19"
+    "lodash": "4.17.21"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/NitorCreations/nflow-explorer.git"
   },
   "devDependencies": {
-    "autoprefixer": "9.7.6",
-    "bower": "1.8.8",
+    "autoprefixer": "9.8.8",
+    "bower": "1.8.13",
     "extend": ">=3.0.2",
-    "grunt": "1.1.0",
-    "grunt-cli": "1.3.2",
+    "grunt": "1.4.1",
+    "grunt-cli": "1.4.3",
     "grunt-concurrent": "3.0.0",
     "grunt-contrib-clean": "2.0.0",
-    "grunt-contrib-concat": "1.0.1",
-    "grunt-contrib-connect": "2.1.0",
+    "grunt-contrib-concat": "2.0.0",
+    "grunt-contrib-connect": "3.0.0",
     "grunt-contrib-copy": "1.0.0",
-    "grunt-contrib-cssmin": "3.0.0",
+    "grunt-contrib-cssmin": "4.0.0",
     "grunt-contrib-htmlmin": "3.1.0",
     "grunt-contrib-imagemin": "4.0.0",
-    "grunt-contrib-jshint": "2.1.0",
-    "grunt-contrib-uglify": "4.0.1",
+    "grunt-contrib-jshint": "3.1.1",
+    "grunt-contrib-uglify": "5.0.1",
     "grunt-contrib-watch": "1.1.0",
     "grunt-filerev": "2.3.1",
-    "grunt-karma": "3.0.2",
+    "grunt-karma": "4.0.2",
     "grunt-newer": "1.3.0",
     "grunt-ng-annotate": "3.0.0",
     "grunt-postcss": "0.9.0",
     "grunt-protractor-runner": "5.0.0",
     "grunt-sass": "3.1.0",
-    "grunt-svgmin": "6.0.0",
+    "grunt-svgmin": "6.0.1",
     "grunt-targethtml": "0.2.6",
     "grunt-usemin": "3.1.1",
     "grunt-wiredep": "3.0.1",
     "jshint-stylish": "2.2.1",
-    "karma": "5.2.2",
-    "karma-coverage": "2.0.2",
-    "karma-jasmine": "3.1.1",
+    "karma": "6.3.10",
+    "karma-coverage": "2.1.0",
+    "karma-jasmine": "4.0.1",
     "karma-junit-reporter": "2.0.1",
     "karma-ng-html2js-preprocessor": "1.0.0",
     "karma-phantomjs-launcher": "1.0.4",
     "karma-sinon": "1.0.5",
-    "karma-spec-reporter": "0.0.32",
+    "karma-spec-reporter": "0.0.33",
     "load-grunt-tasks": "5.1.0",
-    "moment": "2.24.0",
-    "node-sass": "4.14.1",
+    "moment": "2.29.1",
+    "node-sass": "7.0.1",
     "phantomjs-prebuilt": "2.1.16",
     "protractor": "7.0.0",
     "protractor-http-mock": "0.10.0",
-    "selenium-standalone": "6.17.0",
-    "serve-static": "1.14.1",
-    "sinon": "7.5.0",
+    "selenium-standalone": "8.0.8",
+    "serve-static": "1.14.2",
+    "sinon": "12.0.1",
```

Comment:
Starting from version 8.0.0, sinon requires that the browser provides `Map`. The used phantomjs does not, so the latest possible sinon version is 7.5.0.
```
12 01 2022 12:39:41.791:INFO [launcher]: Launching browsers PhantomJS with concurrency unlimited
12 01 2022 12:39:41.794:INFO [launcher]: Starting browser PhantomJS
12 01 2022 12:39:41.922:INFO [PhantomJS 2.1.1 (Linux x86_64)]: Connected on socket DYMLqi69YyPh6lr0AAAB with id 51113872
PhantomJS 2.1.1 (Linux x86_64) ERROR
  ReferenceError: Can't find variable: Map
  at node_modules/sinon/pkg/sinon.js:4115:35

PhantomJS 2.1.1 (Linux x86_64): Executed 0 of 0 ERROR (0.083 secs / 0 secs)
```